src/EditorFeatures/Core/Implementation/Suggestions/SuggestionsOptions.cs

```diff
@@ -10,8 +10,8 @@ internal static class SuggestionsOptions
     {
         private const string FeatureName = "SuggestionsOptions";
 
-        public static readonly Option2<bool?> Asynchronous = new(FeatureName, nameof(Asynchronous), defaultValue: null,
-            new RoamingProfileStorageLocation("TextEditor.Specific.Suggestions.Asynchronous2"));
+        public static readonly Option2<bool> Asynchronous = new(FeatureName, nameof(Asynchronous), defaultValue: true,
+            new RoamingProfileStorageLocation("TextEditor.Specific.Suggestions.Asynchronous3"));
```

Comment:
note: this value needed to change so that if someone in 17.0 disables AsyncLightbulbs that we don't carry that value forever onwards into 17.1.  The profile option exists as a hidden flag if we need to be able to disable through registry.